src/request/types/btcMethods.ts

```diff
@@ -123,3 +123,7 @@ export type SignPsbtResult = {
 };
 
 export type SignPsbt = MethodParamsAndResult<SignPsbtParams, SignPsbtResult>;
+
+export type GetAccountResult = Address[] | Record<string, string> | string[];
```

Comment:
That's a strange mix of potential results 😅 

I'm personally a fan of:
`export type GetAccountResult = Address[];`

Reply:
my bad i actually were exploring the options i actually settled on `Address[]` it's in the PR description

Reply:
Awesome 😄